### PR TITLE
Use file.separator to express difference in FS.

### PR DESCRIPTION
**Pull request by lifehome**

### CODE REVIEW

**Comment by lifehome** [comment]
Give a little bump. (Close PR in 24hrs)

_Ping @Mtihc._

**Comment by lifehome** [comment]
Jeez I didn't make it dead.

File: src/main/java/com/mtihc/regionselfservice/v2/plots/PlotControl.java [hunk 1]
```diff
@@ -33,1336 +33,1179 @@
...
+	    if (region.isOwner(playerName.toLowerCase())) {
+		count++;
+	    }
+	}
+	return count;
+    }
+    
+    private static final HashSet<Byte> invisibleBlocks = new HashSet<Byte>();
+    
+    @SuppressWarnings("deprecation")
```
Comment (Mtihc) [inline src/main/java/com/mtihc/regionselfservice/v2/plots/PlotControl.java]:
```suggestion
```
I would suggest not to suppress "deprecation" warnings. I would either try to use not-deprecated code, or just keep letting it warn us.

File: src/main/java/com/mtihc/regionselfservice/v2/plots/PlotControl.java [hunk 2]
```diff
@@ -33,1336 +33,1179 @@
...
+	    invisibleBlocks.add((byte) Material.STATIONARY_WATER.getId());
+	    invisibleBlocks.add((byte) Material.LAVA.getId());
+	    invisibleBlocks.add((byte) Material.STATIONARY_LAVA.getId());
+	    invisibleBlocks.add((byte) Material.SNOW.getId());
+	    invisibleBlocks.add((byte) Material.LONG_GRASS.getId());
+	}
+	return invisibleBlocks;
+    }
+    
+    @SuppressWarnings("deprecation")
```
Comment (Mtihc) [inline src/main/java/com/mtihc/regionselfservice/v2/plots/PlotControl.java]:
```suggestion
```
I would suggest not to suppress "deprecation" warnings. I would either try to use not-deprecated code, or just keep letting it warn us.

**Review by Mtihc — APPROVED** [review]

**Comment by Mtihc** [comment]
Hi, sorry for the ultra-late response. And thank you for your help. Is there anything else I can help with?

**Comment by lifehome** [comment]
Hmmm I am not sure tho. iirc I actually hired a guy to update this plugin in the past, but me myself got off the game then.

I might consider maintain it for 1.13.1, but since some large updates and backlogs, I may not have enough time to devote for the big update

**Comment by lifehome** [comment]
And hey, no worries for the late reply! not so many PR can live four years X)

**Comment by Mtihc** [comment]
> I might consider maintain it for 1.13.1, but since some large updates and backlogs, I may not have enough time to devote for the big update

Yeah, I don't have the time either.

**Pull request merged**